modules/lib/src/main/resources/assets/js/app/settings/dialog/project/create/data/ProjectParentDialogStepData.ts

```diff
@@ -3,14 +3,18 @@ import {Project} from '../../../../data/project/Project';
 
 export class ProjectParentDialogStepData extends ProjectDialogStepData {
 
-    private parentProject: Project;
+    private parentProjects: Project[];
 
-    setParentProject(value: Project): ProjectParentDialogStepData {
-        this.parentProject = value;
+    setParentProjects(value: Project[]): ProjectParentDialogStepData {
+        this.parentProjects = value;
         return this;
     }
 
-    getParentProject(): Project {
-        return this.parentProject;
+    getParentProjects(): Project[] {
+        return this.parentProjects;
+    }
+
+    getFirstParentProject(): Project {
```

Comment:
Let's rename this to `getMainParentProject`. In the future we might allow user to select the main parent.